tf2onnx/tfonnx.py

```diff
@@ -1255,6 +1255,45 @@ def onehot_op(ctx, node, name, args):
         return
 
 
+def onehot_op9(ctx, node, name, args):
+    # T output = OneHot(uint8/int32/int64 input, T depth, T on-value, T off-value, @int axis, @dtype)
+    # tf requires that dtype is same as on-value's and off-value's dtype
+    # in ONNX, op's schema is (input, depth, [off-value, on-value], @int axis)
```

Comment:
you can probably write that to "T3 output = OneHot(T1 indices, T2 depth, T3 values, @int axis)  # values=[off_value on_value]"

Reply:
done